contracts/Quest.sol

```diff
@@ -66,10 +66,15 @@ contract Quest is ReentrancyGuardUpgradeable, PausableUpgradeable, OwnableUpgrad
         __ReentrancyGuard_init();
     }
 
+    /// @dev The amount of tokens the quest needs to pay all redeemers plus the protocol fee
+    function totalTransferAmount() external view returns (uint256) {
+        return this.maxTotalRewards() + this.maxProtocolReward();
+    }
```

Comment:
I love that you exported this out into its own fx. Easier to just call this than having to calculate all the time